specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/acquisitions.json

```diff
@@ -76,26 +75,10 @@
         "modelAsString": true
       }
     },
-    "AcquisitionProperties": {
-      "description": "The Properties of page BLOB acquisition.",
+    "Acquisition": {
```

Comment:
Is this no longer a resource?  This will have some consequences in PowerShell.

Reply:
Acquisition is a proxy-only resource, do we need to use "x-ms-azure-resource" base class for it?

Reply:
I'll still add a resource tag under it to avoid the side effect in powershell.